src/etc/rc.bootup

```diff
@@ -227,6 +229,9 @@ if (!$debugging) {
 /* re-make hosts file after configuring interfaces */
 system_hosts_generate();
 
+/* add configured trusted CA certificates to /etc/ssl/certs */
+ca_trustedcerts_generate();
+
```

Comment:
Why do trusted certs need to be re/generated during boot up?   /etc/ssl/certs is a static dir is it not?  The certs should still be there across reboots.  Only reason I think of is restoring/swapping a config.  But the trusted certs re/generation for that scenario should and could be done by other means.  Such as part of the config restore process.  Or a trusted certs regeneration button in ca manager for those times when manually swapping a config.